packages/block-editor/src/components/off-canvas-editor/block.js

```diff
@@ -331,10 +332,14 @@ function ListViewBlock( {
 						}
 					>
 						{ () => (
-							<BlockEditButton
-								label={ editAriaLabel }
-								clientId={ clientId }
-							/>
+							<>
+								{ isEditable && (
+									<BlockEditButton
+										label={ editAriaLabel }
+										clientId={ clientId }
+									/>
+								) }
+							</>
```

Comment:
Could we simplify this to not use the `React.Fragment` (`<></>`)? It'll save us a level of indentation and keep things more readable.

Reply:
I'm not clear what @ajlende is recommending here. Let's move forward and merge the PR and we can always follow up when he has time to advise us.